advanced_functionality/autogluon-tabular-containers/AutoGluon_Tabular_SageMaker_Containers.ipynb

```diff
@@ -0,0 +1,735 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    data = df.drop(columns="class")[:100].values
</pre><p>What is the <code>[:100]</code>? Are you predicting on 100 items? If so, you could note that for clarity.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2957/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>Yes, it is 100 items. Added text explaining that.</p>